tests_integration/conftest.py

```diff
@@ -105,3 +117,27 @@ def invoke_integration_without_format(self, *args, **kwargs) -> CommandResult:
 @pytest.fixture
 def runner(test_snowcli_config_provider):
     return SnowCLIRunner(app, test_snowcli_config_provider)
+
+
+class QueryResultJsonEncoderError(RuntimeError):
+    def __init__(self, output: str):
+        super().__init__(f"Can not parse query result:\n{output}")
+
+
+@pytest.fixture
+def project_file(temporary_working_directory, test_root_path):
```

Comment:
```suggestion
def project_directory_copy(temporary_working_directory, test_root_path):
```

Reply:
`project_directory`?

Reply:
`project_directory` could be a good name but I saw the following code:

```python
with project_file("example_streamlit") as pdir:
        (pdir / "environment.yml").unlink()
        shutil.rmtree(pdir / "pages")
```

And the first thing I thought about was: do you remove files from some subdirectory of `tests` directory?
I can agree for `project_directory` but I just wanted to tell you what I was afraid of when reading the code without a name suggesting that it's a copy/temp dir.

Reply:
`tmp_project_directory`?

Reply:
We agreed that we can stay with `project_directory`.

Reply:
We decided on `project_directory`